### ISSUE TITLE
Failed to set the innerHTML property on Element

### ISSUE THREAD

**Issue by flashlight4** [globaleaks/globaleaks-whistleblowing-software#4350]
### What version of GlobaLeaks are you using?

5.0.33

### What browser(s) are you seeing the problem on?

Chrome

### What operating system(s) are you seeing the problem on?

macOS

## Description
After updating from version 5.0.26 to 5.0.33, the Markdown-formatted introduction text intended to display on the homepage is no longer visible. An error is logged in the browser console.

## Steps to Reproduce
	1.	Update the application from version 5.0.26 to 5.0.33.
	2.	Open the application homepage.
	3.	Observe that the introduction text is missing.
	4.	Check the browser console for errors.

## Expected Behavior
The Markdown-formatted introduction text should render properly on the homepage as it did in version 5.0.26.

## Observed Behavior
The introduction text does not appear. The following error is logged in the browser console:

```
core.mjs:7404 ERROR TypeError: Failed to set the 'innerHTML' property on 'Element': This document requires 'TrustedHTML' assignment.
    at ngx-markdown.mjs:537:36
    at Generator.next (<anonymous>)
    at i (asyncToGenerator.js:3:1)
    at ve (asyncToGenerator.js:17:1)
    at u.invoke (zone.js:369:28)
    at Object.onInvoke (core.mjs:7251:33)
    at u.invoke (zone.js:368:34)
    at N.run (zone.js:111:43)
    at zone.js:2538:40
    at u.invokeTask (zone.js:402:33)
handleError @ core.mjs:7404
```

## Environment Details
	•	Browser: Brave [Version 1.73.91 Chromium: 131.0.6778.85 (Official Build) (arm64)]
	•	Operating System: macOS Sequoia 15.1.1
	•	Application Version: 5.0.33 (works correctly in 5.0.26)

## Additional Context
The issue seems related to security enhancements introduced in the new version of Chromium or the application. Specifically, the error suggests that the application requires TrustedHTML for innerHTML assignments, potentially due to stricter content security policies.

## Possible Related Changes
	•	Update to ngx-markdown handling of HTML.
	•	Changes in Chromium’s handling of innerHTML.

## Suggested Fix
Ensure Markdown rendering respects the new TrustedHTML requirement or implement a fallback mechanism compatible with the latest version of Chromium.

Thank you for investigating this issue!

### Proposed solution

Ensure Markdown rendering respects the new TrustedHTML requirement or implement a fallback mechanism compatible with the latest version of Chromium.

**Comment by evilaliv3** [comment]
Thank you @flashlight4 .

We should have fixed this in 5.0.34

Would you please retry?

**Comment by flashlight4** [comment]
Thank you very much @evilaliv3 , 

it just works fine again with 5.0.35.
